src/js/components/PopoverComponent.js

```diff
@@ -0,0 +1,81 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var Util = require("../helpers/Util");
+
+const DEFAULT_ALIGNED = "default";
```

Comment:
Yeah, what I meant is: why aren't they props? Not necessary now, but in the future we might want to predetermine the alignment from "outside" (imagine reusing this for the labels dropdown). Anyway, not a blocker.